src/app/components/Students.js

```diff
@@ -1,5 +1,23 @@
 import React from 'react';
+import { connect } from 'react-redux';
+import getStudentsData from '../store/actions/getStudentsData';
 
-const Students = () => <h1>Students</h1>;
+class Students extends React.Component {
+  componentDidMount() {
+    const { getStudentsData } = this.props;  // eslint-disable-line
+    getStudentsData();
+  }
 
-export default Students;
+  render() {
+    return <h1>Students</h1>;
+  }
+}
+
+const mapStateToProps = state => ({ students: state.students.data });
```

Comment:
i think it should be:
```suggestion
const mapStateToProps = state => ({ students: state.students.studentsData });
```
:)

Reply:
Awesome catch! Thanks a lot 👍